### DISCUSSION TITLE
download oracle database for linux 19c

### DISCUSSION THREAD

**Discussion by thelllmike** [community/community#68907] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://www.oracle.com/dz/database/technologies/oracle19c-linux-downloads.html

when i try to log to oracle keep showing error (username password correct) anyone have idea?

**Comment by Spixmacaw** [comment]
Downloading Oracle Database 19c for Linux involves several steps. Here's a high-level overview of the process:

1. **Prerequisites**:
   - Ensure that your Linux system meets the minimum requirements for Oracle Database 19c.
   - Make sure you have sufficient disk space, typically several gigabytes.

2. **Oracle Account**:
   - You'll need an Oracle account to download the software. If you don't have one, you can create it for free on the Oracle website.

3. **Download Oracle Database 19c**:

   a. Go to the Oracle Database Downloads page: [Oracle Database Downloads](https://www.oracle.com/database/technologies/)

   b. Accept the Oracle Technology Network License Agreement.

   c. Scroll down to the "Oracle Database 19c" section and select the "See All" link.

   d. In the list of available downloads, locate the "Oracle Database 19c for Linux" section.

   e. Choose the appropriate download file for your Linux distribution. There are RPM and ZIP files available. RPM files are typically easier to install on RPM-based distributions (e.g., Oracle Linux, Red Hat, CentOS), while ZIP files provide more flexibility but require manual installation.

   f. Click the download link to start the download. You may be prompted to log in with your Oracle account if you haven't already.

4. **Install Oracle Database 19c**:
   
   a. **Using RPM Package (Oracle Linux, Red Hat, CentOS)**:
      - If you downloaded an RPM file, you can install it using the `yum` or `dnf` package manager, depending on your Linux distribution.
      - Open a terminal and navigate to the directory where the RPM file is located.
      - Run the following command (replace `oracle-database-xxxx-xxxx.rpm` with the actual RPM file name):

     ```bash
     sudo yum localinstall oracle-database-xxxx-xxxx.rpm
     ```

   b. **Using ZIP File (Any Linux Distribution)**:
      - If you downloaded a ZIP file, you'll need to manually extract and install Oracle Database.
      - Extract the ZIP file to a directory of your choice. You can use the `unzip` command:

     ```bash
     unzip oracle-database-xxxx-xxxx.zip
     ```

      - Follow the installation instructions provided in the Oracle documentation. Typically, you'll need to run the Oracle Universal Installer (OUI) and go through a guided installation process.

5. **Configuration and Post-Installation**:
   - After the installation is complete, follow the post-installation steps in the Oracle documentation to configure your database and perform any necessary setup, including creating a database instance, setting up environment variables, and configuring user accounts.

Please note that Oracle's licensing terms may change, and it's essential to review and comply with their current licensing agreements and requirements when downloading and using Oracle Database. Additionally, Oracle Database installations can be complex, so it's advisable to refer to the official Oracle documentation for detailed installation and configuration instructions specific to your Linux distribution and use case.

**Comment by thelllmike** [comment]
Ensure Correct SID and Service Name
Double-check the SID or Service Name you're trying to connect to. Oracle can sometimes show login errors if you're not connecting to the correct database instance. Ensure the correct hostname, port, and SID are configured in your connection.
